docs/book/src/security-guidelines/index.md

```diff
@@ -0,0 +1,6 @@
+# Security Guidelines
```

Comment:
nit, not blocking 
I think we can drop `-guidelines` from the folder name (for brevity)

Reply:
Valid point :-) renamed the directory